flake.nix

```diff
@@ -297,6 +305,7 @@
       packages = perSystem (system:
         self.hsFlake.${system}.packages
         // (psProjectFor (nixpkgsFor system)).packages
+        // inputs.kupo-nixos.outputs.defaultPackage.${system}
```

Comment:
This is a derivation, so `packages.x86_64-linux` is a derivation so `hydraJobs.x86_64-linux` is a derivation. An attrset is needed instead.